code/network/multi_observer.cpp

```diff
@@ -59,7 +59,7 @@ void multi_obs_create_player(int player_num,char *name,net_addr *addr,player *pl
 	Net_players[player_num].last_heard_time = timer_get_fixed_seconds();
 	Net_players[player_num].reliable_socket = PSNET_INVALID_SOCKET;
 	Net_players[player_num].s_info.kick_timestamp = -1;
```

Comment:
I'd say `kick_timestamp` could be converted too

Reply:
Done